wfl/DBManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using wfl.Models;
+
+namespace wfl
+{
+    public class DBManager
```

Comment:
I kind of mention this in a comment in `Restaurant.cs` but you ideally want to encapsulate any of the database specific code in a manor that allows you to easily change database implementations in the future. A common practice is to define an interface for classes like this DBManager and have a "DBManager Provider" that uses configuration or dependency injection (or simply hard-codes it) to create/return implementations like this.